texture_manager.c

```diff
@@ -31,44 +31,23 @@
 #include "core/deps/jansson/jansson.h"
 #include "core/platform/threads.h"
 
-// This mode allows the texture manager to opportunistically free canvases
-// if they have not been accessed lately
-#define FREE_CANVASES
-
-// This mode notifies immediately on losing the canvas, otherwise it notifies
-// when the canvas is used
-//#define FREE_CANVASES_NOTIFY_IMMEDIATELY
-
-// 1024 x 1024 with 4 channels
-// this is not necassarily the size of every sheet
-#define MAX_SHEET_BYTES (1024*1024*4)
 #define DEFAULT_SHEET_DIMENSION 64
 
-// Constants for Android phones and starting-points for iOS
-#define MAX_BYTES_FOR_HALFSIZED_TEXTURES 500000000  /* 500 MB formerly 30 MB */
-#define MAX_BYTES_FOR_FULLSIZED_TEXTURES 500000000  /* 500 MB formerly 100 MB */
-
-// Does not use fullsized textures under this memory value
-#define MIN_BYTES_FOR_FULLSIZE_TEXTURES 30000000    /* 30 MB */
+// Large number to start texture memory limit
+#define MAX_BYTES_FOR_TEXTURES 500000000    /* 500 MB */
```

Comment:
Since this supports ratcheting upwards, maybe we should start with 100MB and let it grow as needed (assuming all else is equivalent).